tscreen.go

```diff
@@ -1484,20 +1486,29 @@ func (t *tScreen) mainLoop(stopQ chan struct{}) {
 }
 
 func (t *tScreen) inputLoop(stopQ chan struct{}) {
-
 	defer t.wg.Done()
+	var (
+		n   int
+		err error
+	)
 	for {
 		select {
 		case <-stopQ:
 			return
 		default:
 		}
+
+		err = t.in.SetReadDeadline(time.Now().Add(250 * time.Millisecond))
```

Comment:
Well, 250ms feels like the delay I was having.

Reply:
This causes us to wake up every 250ms.  Which is vile on some platforms.  I was hoping to avoid period polling.

Reply:
...Okay?  What do you propose instead, considering the issues regarding suspend and resume (#452)?